docs/guides/tutorials/cloudflare_workers.rst

```diff
@@ -0,0 +1,258 @@
+
+.. code-block:: bash
+
+    $ edgedb
+
+**Install The EdgeDB npm package**
+
+.. code-block:: bash
+
+    $ npm install edgedb # or pnpm, yarn, bun
+
+**Extend The Default Schema (Optional)**
+
+You can extend the default schema, :code:`dbschema/default.esdl`, to define 
+your data model, and then try it out in the Cloudflare Worker code.
+
+Add new types to the schema file:
+
+.. code-block:: sdl
+
+    module default {
+      type Movie {
+        required title: str {
+          constraint exclusive;
+        };
+        multi actors: Person;
+      }
+
+      type Person {
+        required name: str;
+      }
+    }
+
+Then apply the schema schema to your EdgeDB instance:
+
+.. code-block:: bash
+
+    $ edgedb migration create
+    $ edgedb migrate
+
+Using EdgeDB in a Cloudflare Worker
+====================================
+
+Open the :code:`index.ts` file from the :code:`src` directory in your project, 
+and remove the default code.
+
+To interact with your **local EdgeDB instance**, use the following code:
+
+.. code-block:: typescript
+
+    import * as edgedb from "edgedb";
+
+    export default {
+      async fetch(
+        _request: Request,
+        env: Env,
+        ctx: ExecutionContext,
+      ): Promise<Response> {
+        const client = edgedb.createHttpClient({
+          tlsSecurity: "insecure",
+          dsn: "<your-edgedb-dsn>",
+        });
+        const movies = await client.query(`select Movie { title }`);
+        return new Response(JSON.stringify(movies, null, 2), {
+          headers: {
+            "content-type": "application/json;charset=UTF-8",
+          },
+        });
+      },
+    } satisfies ExportedHandler<Env>;
+
+
+.. note:: EdgeDB DSN
+
+    Replace :code:`<your-edgedb-dsn>` with your EdgeDB DSN.
+    You can obtain your EdgeDB DSN from the command line by running:
+
+    .. code-block:: bash
+
+        $ edgedb instance credentials --insecure-dsn
+
+.. note:: tlsSecurity
+
+    The :code:`tlsSecurity` option is set to :code:`insecure` to allow 
+    connections to a local EdgeDB instance. This lets you test your
+    Cloudflare Worker locally. **Don't use this option in production.**
+
+**Client Setup with EdgeDB Cloud**
+
+If you're using EdgeDB Cloud, you can instead use the following code to 
+set up the client:
+
+.. code-block:: typescript
+
+   const client = edgedb.createHttpClient({
+     instanceName: env.EDGEDB_INSTANCE,
+     secretKey: env.EDGEDB_SECRET_KEY,
+   });
+
+You will need to set the :code:`EDGEDB_INSTANCE` and :code:`EDGEDB_SECRET` 
+environment variables in your Cloudflare Worker project.
+
+Add the following to your :code:`wrangler.toml` file:
+
+.. code-block:: toml
+
+    [vars]
+    EDGEDB_INSTANCE = "your-edgedb-instance"
+    EDGEDB_SECRET_KEY = "your-edgedb-secret-key"
+
+Next, you can run :code:`wrangler types` to generate the types for your
+environment variables.
+
+You can obtain these values from the EdgeDB Cloud dashboard.
```

Comment:
I'd recommend moving this up as a note, just under the first case where the user will need the values.